src/main/java/kr/motd/maven/os/Detector.java

```diff
@@ -64,6 +72,12 @@ protected void detect(Properties props, List<String> classifierWithLikes) {
         setProperty(props, DETECTED_NAME, detectedName);
         setProperty(props, DETECTED_ARCH, detectedArch);
 
+        Matcher versionMatcher = VERSION_REGEX.matcher(osVersion);
+        if (versionMatcher.matches()) {
+            setProperty(props, DETECTED_VERSION_MAJOR, versionMatcher.group(1));
+            setProperty(props, DETECTED_VERSION_MINOR, versionMatcher.group(2));
```

Comment:
Could we also make `DETECTED_VERSION` public and set the full version string? (please update the documentation as well)